repository/SmalltalkCI-Pharo-Core.package/SmalltalkCIPharo9.class/instance/preLoad.st

```diff
@@ -0,0 +1,10 @@
+executing
+preLoad
+	[ (IceRepositoryCreator new
+		location: projectDirectory asFileReference;
+		createRepository) register. ] 
+		on: Error 
+		do: [ :e |
+		      	 self class printWarningToStdout: 'Could not register the tested repository with Iceberg.'.
+		      	 self class printWarningToStdout: 'If you need to use Iceberg features in the CI, see github.com/hpi-swa/setup-smalltalkCI'.
```

Comment:
This somewhat assumes that smalltalkCI runs on GHA, which is a common but not the only case.

Reply:
Are the yml different when not on GHA? I'm not familiar with such runs

Reply:
Yes, the main README.md has some examples: https://github.com/hpi-swa/smalltalkCI#gitlab-ci-template.

Reply:
Would you be okay if that said that is only valid for GHA ?
Don't really know whether that'd work for travis et al.

Reply:
I'd add to the note that the full Git history must be available for Iceberg and then link to the setup-smalltalkCI docs.